File: app/model/EnrichedModelsDTO.scala
package model

import play.api.libs.json._

/**
  * Список моделей товаров.
  * @param models Список моделей товаров.
  */
@javax.annotation.Generated(value = Array("org.openapitools.codegen.languages.ScalaPlayFrameworkServerCodegen"), date = "2024-09-09T22:33:09.055496365Z[Etc/UTC]", comments = "Generator version: 7.8.0")
case class EnrichedModelsDTO(
  models: List[EnrichedModelDTO]
)

object EnrichedModelsDTO {
  implicit lazy val enrichedModelsDTOJsonFormat: Format[EnrichedModelsDTO] = Json.format[EnrichedModelsDTO]
}

File: ympa_fsharp_giraffe_server/src/model/EnrichedModelsDTO.fs
namespace ympa_fsharp_giraffe_server.Model

open System
open System.Collections.Generic
open ympa_fsharp_giraffe_server.Model.EnrichedModelDTO

module EnrichedModelsDTO =

  //#region EnrichedModelsDTO


  type EnrichedModelsDTO = {
    Models : EnrichedModelDTO[];
  }
  //#endregion
